test/models/friend_request_test.rb

```diff
@@ -0,0 +1,41 @@
+require 'test_helper'
+
+class FriendRequestTest < ActiveSupport::TestCase
```

Comment:
Great job too on the FriendRequestTest! You tested validations and all associations/instance methods affecting FriendRequest's relationship with user and friendship